### DISCUSSION TITLE
域名绑定不成功

### DISCUSSION THREAD

**Discussion by yl1010code** [walinejs/waline#594] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/71326380/136577790-7fc9e9ec-4272-4fd8-9f46-edb48897edfc.png)
Error:The domain yl1010code.gitee.io is already owned by another Vercel account.

不知道我还有其他的账号，而且还绑定了域名

**Comment by lizheming** [comment, marked as answer]
gitee.io 这域名你没办法做 A 记录吧，应该是不支持绑定的

**Comment by ghost** [comment]
> gitee.io 这域名你没办法做 A 记录吧，应该是不支持绑定的

是不是只能用自己网站的域名，而且不带二级域名的？

**Reply by Mister-Hope** [reply]
不，你需要一个可以做CNAME解析的域名，其他的无所谓
